### DISCUSSION TITLE
erreur DEL pour l'ID

### DISCUSSION THREAD

**Discussion by mxtla** [danymat/INSAnonym-utils#62] [Q&A, Bug report & Support] [answered]
Bonjour, 

J'ai mis la valeur DEL pour certains IDs comme autorisé dans les règles. Cependant, lorsque je soumets mon fichier, j'ai une erreur.
Comment puis-je la régler ? 

Margaux

**Comment by beng-git** [comment, marked as answer]
Bonjour, 

Pouvez-vous préciser quel est le fichier en cause (nom de l'équipe & du fichier ?)

Cordialement,

BN

**Reply by mxtla** [reply]
Bonjour, 

#47 test
Equipe PrivUQAM

Merci

**Reply by beng-git** [reply]
et la #51 ?

**Comment by danymat** [comment]
Bonsoir, après analyse de votre soumission `#51`, certaines lignes ont l'identifiant supprimé sans forcément de DEL à la place. Je reviens vers vous concernant la `#47`

**Reply by mxtla** [reply]
Oui pour la #51 c'était un autre test, je me doutais que ca ne fonctionnerait pas.

Merci

**Reply by beng-git** [reply]
Si jamais c'est nécessaire et qu'on fait un fix ce soir, nous pourrons faire une petite extension de deadline pour que vous puissiez soumettre votre fichier.

**Reply by danymat** [reply]
Concernant l soumission `#47, votre fichier fait +54M de lignes. Faites attention, on ne peut pas modifier le nombre de lignes, comme expliqué dans la doc user :  

```md
- It is forbidden to change the number of lines and columns. Your submission is automatically randomized in the server.
```

**Reply by danymat** [reply]
N'hésitez pas à mettre la réponse comme réponse acceptée si elle vous satisfait !
